addons/account/models/account_payment.py

```diff
@@ -383,7 +383,7 @@ def create_payments(self):
 
 class account_payment(models.Model):
     _name = "account.payment"
-    _inherit = ['mail.thread', 'account.abstract.payment']
+    _inherit = ['mail.thread', 'account.abstract.payment', 'mail.activity.mixin']
```

Comment:
I would prefer to have 'mail.activity.mixin just after mail.thread except if you have a good reason to keep this in this order.
['mail.thread, 'mail.activity.mixin', 'account.abstract.payment']